### ISSUE TITLE
Airgap tests files structure doesn't currently support multiple versions

### ISSUE THREAD

**Issue by kimwnasptd** [canonical/bundle-kubeflow#1054]
### Bug Description

The [current](https://github.com/canonical/bundle-kubeflow/tree/9db4dd10a28d39b575fb1ab8f768097ca9e2cc37) structure of the airgap tests is the following:
```
tests/
  airgapped/
    knative/
    pipelines/
    training/
    lxc/
    airgap.sh
    ckf.sh
``` 

Since the repo doesn't have dedicated branches https://github.com/canonical/bundle-kubeflow/issues/743 we can't currently have files for both 1.8 and 1.9 releases for airgapped. This came up also during the discussion in https://github.com/canonical/bundle-kubeflow/pull/1038#discussion_r1743540218

Since we have to use the same branch for the different releases, we'll need to have folders for each release's files.

### To Reproduce

Try to run airgap tests for 1.9 and 1.8

### Environment

Repo from commit 9db4dd10a28d39b575fb1ab8f768097ca9e2cc37

### Relevant Log Output

```shell
<none>
```

### Additional Context

A proposed structure was the following https://github.com/canonical/bundle-kubeflow/pull/1038#discussion_r1744495995:
```bash
tests/
  airgapped/
    1.8/
      pipelines/
      training/
      testing-images.txt
      deploy.sh  # copied from scripts/airgapped/deploy-1.8.sh
    1.9/
      pipelines/
...
```

@mvlassis also proposed to unify the two `deploy-1.8.sh` and `deploy-1.9.sh` scripts into one that could work on both releases

**Comment by mvlassis** [comment]
@kimwnasptd This issue can be separated into two:
- Introduce a file structure for testing airgapped on multiple versions (1.8, 1.9). This was handled and merged in #1053.
- Merge the scripts `deploy-1.8.sh` and `deploy-1.9.sh` into one. This hasn't been done yet.

I am closing this issue and creating one for the task of merging the 2 deployment scripts.

**Comment by mvlassis** [comment]
I have created #1083 for tracking the merging of the 2 deployment scripts.
